test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_With_Fake_Parameter.lua

```diff
@@ -0,0 +1,103 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [HMI RPC validation]: SDL behavior: HMI sends request (response, notification) with fake parameters that SDL should transfer to mobile
```

Comment:
@OVikhrov, please update requirement summary with CRQ related to OnSeekMediaClockTimer also